### PR TITLE
Adding Malayalam Locale on locales folder

### PR DESCRIPTION
**Pull request by ranjithsiji**
I translated the locale to Malayalam Language. Plz check the translation.

### CODE REVIEW

File: locales/ml-IN.ftl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title = ഫയര്‍ഫോക്സ് ക്വാണ്ടം റിലീസ് പ്രചാരണപ്രവര്‍ത്തനങ്ങള്‍
+project-name = ഫയര്‍ഫോക്സ് ക്വാണ്ടം റിലീസ്
+main-title = =ഫയര്‍ഫോക്സ് തിരിച്ചടിക്കുന്നു!
+main-description = പുതിയ ഫയര്‍ഫോക്സിനെപ്പറ്റിയുള്ള വിവരങ്ങള്‍ പ്രചരിപ്പിക്കുവാന്‍ ഞങ്ങളെ സഹായിക്കുക! 14-ാം തീയ്യതി ചൊവ്വാഴ്ച 14:00 യുടിസി സമയത്ത് പുതിയ ഫയര്‍ഫോക്സില്‍ എന്താണ് നിങ്ങള്‍ക്ക് ഇഷ്ടപ്പെട്ടത് എന്ന് #NewFirefox എന്ന ഹാഷ്ടാഗില്‍ ട്വീറ്റ് ചെയ്യുക അല്ലെങ്കില്‍ ഔദ്യോഗിക ഫയര്‍ഫോക്സിന്റെ ട്വീറ്റുകള്‍ റീട്വീറ്റ് ചെയ്യുക
+thunderclap-cta = ഫയര്‍ഫോക്സ് ക്വാണ്ടത്തിനെ പിന്‍തുണക്കുക!
+thunderclap-link = https://www.thunderclap.it/projects/64595-firefox-quantum-launch-day
+official-tweet-cta = ഔദ്യോഗിക ട്വീറ്റുകള്‍ റീട്വീറ്റ് ചെയ്യുക
+tweet-1 = #NewFirefox 🦊 പുതിയ ഫയര്‍ഫോക്സ് ഇന്ന് പുറത്തിറങ്ങി. ഇത് നേരത്തേയുള്ളതിനേക്കാളും രണ്ടിരട്ടി വേഗതകൂടിയതാണ്. 💥 ഡൌണ്‍ലോഡ് ചെയ്യുക :https://mzl.la/newfirefox
```
Comment (MichaelKohler) [inline locales/ml-IN.ftl]: there seems to be a space missing before the URL. Is that on purpose?

File: locales/ml-IN.ftl [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+tweet-2 = പുതിയ ഫയര്‍ഫോക്സ് ഒരു ___________ നേക്കാള്‍ വേഗതയുള്ളതാണ്. നന്ദി @Firefox!
+tweet-3 = പുതിയ ഫയര്‍ഫോക്സ് പുറത്തിറങ്ങി. അത് വളരെ വേഗതയേറിയതാണ്. ഡൌണ്‍ലോഡ് ചെയ്യൂ : https://mzl.la/newfirefox
```
Comment (MichaelKohler) [inline locales/ml-IN.ftl]: I'm missing the hashtag here :)

File: locales/ml-IN.ftl [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+hashtag = #NewFirefox
+github = ഗിറ്റ്ഹബ്ബ്
+legal = നിയമപരം
+license = അനുമതിപത്രം
+guidelines = മാര്‍ഗനിര്‍ദ്ദേശകരേഖകള്‍
+mozilla-description =
+      മോസില്ല ഒരു ആഗോള ലാഭേച്ഛയില്ലാതെ പ്രവര്‍ത്തിക്കുന്ന ഒരു ആഗോള സംഘടനയാണ്. അത് നിങ്ങളുടെ ഓണ്‍ലൈന്‍ അനുഭവത്തെ നിങ്ങള്‍ക്ക് നിയന്ത്രിക്കാന്‍ സാധിക്കുന്നതാക്കുന്നു. കൂടാതെ പൊതുനന്മക്കായുള്ള ഭാവി വെബ്    
+      നിര്‍മ്മിക്കുന്നതില്‍ പ്രധാന പങ്കുവഹിക്കുന്നു. വെബ്‍സൈറ്റ്
+  <a href="https://mozilla.org" target="_blank">mozilla.org</a>
+official-tweet-id = tbd
```
Comment (MichaelKohler) [inline locales/ml-IN.ftl]: can you please add the same ID as in the en-US file?

**Review by MichaelKohler — COMMENTED** [review]
Thanks. Please see my comments.

File: locales/ml-IN.ftl (hunk 1 above)
Comment (ranjithsiji) [reply locales/ml-IN.ftl]: corrected that

**Comment by ranjithsiji** [comment]
Hope the corrections are ok.

**Pull request closed without merging**